api/pkg/inference-logger/logger/mlobs_sink.go

```diff
@@ -6,6 +6,8 @@ import (
 	"errors"
 	"fmt"
 
+	"golang.org/x/exp/rand"
```

Comment:
Why are you using `golang.org/x/exp/rand` instead of `math/rand` package? To my knowledge, the latter does not need to run `seed` first.

Reply:
No particular reason actually, i wasn't aware of the math/rand package. I will checkout the math/rand package,